File: Level 0/코딩 기초 트레이닝/코드 처리하기/solution.py
def solution(code):
    answer = ""
    mode = 0

    for index in range(len(code)):
        if code[index] == "1":
            mode = 0 if mode else 1
            continue

        if index % 2 == mode:
            answer += code[index]

    return answer if len(answer) else "EMPTY"

File: Level 0/코딩 기초 트레이닝/코드 처리하기/solution.java
class Solution {
    public String solution(String code) {
        String answer = "";
        int mode = 0;

        for (int i = 0; i < code.length(); i++) {
            char current = code.charAt(i);

            if (current == '1') {
                mode = mode == 0 ? 1 : 0;
                continue;
            }

            if (i % 2 == mode) {
                answer += code.charAt(i);
            }
        }

        return answer.length() == 0 ? "EMPTY" : answer;
    }
}